PowerUp/app/src/main/res/layout/intro_screen_4.xml

```diff
@@ -0,0 +1,16 @@
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" tools:context="powerup.systers.com/IntroActivity"
+    android:background="@drawable/screenshot">
```

Comment:
You need to also update a version of this image for horizontal displays using qualifiers